model/src/pyrenew/convolve.py

```diff
@@ -17,62 +17,110 @@
 
 import jax.numpy as jnp
 from jax.typing import ArrayLike
+from pyrenew.transformation import IdentityTransform
```

Comment:
Question: isn't this patter better?

```suggestion
from pyrenew.transformation.builtin import IdentityTransform
```

Reply:
No, that's not better, because the transform module is imported via `__init__.py`. That said, I'm still a bit puzzled. Perhaps this is better:

```suggestion
from numpyro.distributions.transforms import IdentityTransform
```

Reply:
I think it is fine as is.

Reply:
I agree, I think if we're going to wrap `numpyro.distributions.transforms` we can and should internally refer to it through our wrapper.